app/models/institution_builder.rb

```diff
@@ -310,6 +311,18 @@ def self.add_scorecard(version_id)
 
     Institution.connection.update(str)
   end
+  
+  def self.update_lat_lon_from_scorecard(version_id)
```

Comment:
This is not needed as column is set from scorecard at https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/institution_builder.rb#L303